app/frontend/src/features/search/SearchResultsList.tsx

```diff
@@ -44,11 +49,17 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   searchResult: {
-    [theme.breakpoints.down("sm")]: {
-      maxWidth: "80%",
-    },
-    [theme.breakpoints.up("md")]: {
-      margin: theme.spacing(2),
+    marginBottom: theme.spacing(3),
+    borderRadius: theme.shape.borderRadius * 2,
+    backgroundColor: theme.palette.primary.contrastText,
+    boxShadow: "0 0 4px rgba(0,0,0,0.25)",
+    [theme.breakpoints.down("md")]: {
+      "& > div > *": {
+        padding: 0,
+        margin: theme.spacing(2),
+        height: "calc(100% - 2rem)",
+        overflow: "hidden",
+      },
```

Comment:
I don't understand this - there doesn't seem to be any div directly under the search result.

Reply:
Don't know what happened as I had it working, but you're right. Switched it to button and it works.

Reply:
I see. Can you use the MUI class instead of this selector - it's clearer what you're targeting. So it would be `"& MuiCardContent-root:`